stdlib/LibGit2/src/types.jl

```diff
@@ -206,6 +204,27 @@ abstract type Payload end
     payload::Ptr{Cvoid}
 end
 
+@kwdef struct RemotePayloads
+    sideband_progress::Ptr{Cvoid}
+    completion::Ptr{Cvoid}
+    credentials::Ptr{Cvoid}
+    certificate_check::Ptr{Cvoid}
+    transfer_progress::Ptr{Cvoid}
+    update_tips::Ptr{Cvoid}
+    pack_progress::Ptr{Cvoid}
+    push_transfer_progress::Ptr{Cvoid}
+    push_update_reference::Ptr{Cvoid}
+    push_negotiation::Ptr{Cvoid}
+    transport::Ptr{Cvoid}
```

Comment:
These are Julia handles. They should not be typed as opaque C handles. It looks like the input is a `Dict`, so `RemotePayloads` can also just be a `Dict{Symbol, Any}`.

Reply:
Should I use `::Any` or `Ref{Any}`. We could do either.